packages/fxa-content-server/app/tests/spec/models/account.js

```diff
@@ -2177,238 +2173,155 @@ describe('models/account', function() {
     });
   });
 
-  describe('fetchDevices', function() {
-    beforeEach(function() {
-      account.set('sessionToken', SESSION_TOKEN);
-
-      sinon.stub(fxaClient, 'deviceList').callsFake(function() {
-        return Promise.resolve([
-          {
-            id: 'device-1',
-            isCurrentDevice: false,
-            name: 'alpha',
-          },
-          {
-            id: 'device-2',
-            isCurrentDevice: true,
-            name: 'beta',
-          },
-        ]);
-      });
-    });
-
-    it('fetches the device list from the back end', function() {
-      return account.fetchDevices().then(result => {
-        assert.isTrue(fxaClient.deviceList.calledWith(SESSION_TOKEN));
-        assert.equal(result.length, 2);
-        assert.equal(result[0].clientType, 'device');
-        assert.equal(result[0].name, 'alpha');
-      });
-    });
-  });
-
-  describe('fetchSessions', function() {
+  describe('fetchAttachedClients', function() {
     beforeEach(() => {
       account.set('sessionToken', SESSION_TOKEN);
 
-      sinon.stub(fxaClient, 'sessions').callsFake(() => {
+      sinon.stub(fxaClient, 'attachedClients').callsFake(() => {
         return Promise.resolve([
           {
-            deviceName: 'alpha',
+            deviceId: 'device-1',
             deviceType: 'desktop',
-            id: 'device-1',
-            isCurrentDevice: false,
-            isDevice: true,
+            isCurrentSession: false,
+            name: 'alpha',
+            sessionTokenId: 'session-1',
           },
           {
-            id: 'foo',
-            isCurrentDevice: false,
+            isCurrentSession: false,
             name: 'session',
+            sessionTokenId: 'foo',
           },
           {
-            deviceName: 'beta',
+            deviceId: 'device-2',
             deviceType: 'mobile',
-            id: 'device-2',
-            isCurrentDevice: true,
-            isDevice: true,
+            isCurrentSession: true,
+            name: 'beta',
+            sessionTokenId: 'session-2',
           },
         ]);
       });
     });
 
-    it('fetches the session list from the back end', function() {
-      return account.fetchSessions().then(result => {
-        assert.isTrue(fxaClient.sessions.calledWith(SESSION_TOKEN));
+    it('fetches the list of attached clients from the back end', function() {
+      return account.fetchAttachedClients().then(result => {
+        assert.isTrue(fxaClient.attachedClients.calledWith(SESSION_TOKEN));
         assert.equal(result.length, 3);
-        assert.equal(result[0].clientType, 'device');
         assert.equal(result[0].name, 'alpha');
-        assert.equal(result[0].type, 'desktop');
         assert.equal(result[0].deviceType, 'desktop');
-        assert.ok(result[0].isDevice);
-        assert.notOk(result[0].isWebSession);
 
-        assert.equal(result[1].clientType, 'webSession');
         assert.equal(result[1].name, 'session');
```

Comment:
Does anything else need to be asserted for result[1] for completeness? Do any conversions take place that should be checked?